src/qt_gui/gui_context_menus.h

```diff
@@ -165,12 +165,12 @@ class GuiContextMenus : public QObject {
                     if (createShortcutLinux(linkPath, ebootPath, iconPath)) {
 #endif
                         QMessageBox::information(
-                            nullptr, "Shortcut Creation",
-                            QString("Shortcut created successfully:\n %1").arg(linkPath));
+                            nullptr, "Shortcut creation...",
+                            QString("Shortcut created successfully !\n %1").arg(linkPath));
```

Comment:
```suggestion
                            QString("Shortcut created successfully!\n %1").arg(linkPath));
```
Having a space before the exclamation mark is a grammatical error